platform/solr/solr-factory/src/main/java/org/codice/solr/factory/impl/ConfigurationFileProxy.java

```diff
@@ -31,9 +31,11 @@
 import org.slf4j.LoggerFactory;
 
 /**
- * Abstraction layer for accessing files or directories on disk. Provides different implementations
- * depending on if the code is run within an OSGi container or not.
+ * @deprecated Intrigue and simple search replace this functionality. Abstraction layer for
```

Comment:
✏️ We should not be referencing Catalog UI's like Intrigue at a platform level.

Reply:
fixed to more generalized